code/modules/mob/living/carbon/alien/diona/diona_powers.dm

```diff
@@ -299,45 +283,44 @@
 	for(var/mob/living/carbon/alien/diona/D in view(7))
 		if(D.master_nymph == src && mind && !client) //if the nymph is subservient to you
 			mind.transfer_to(D)
-			D.stunned = 0//Switching mind seems to temporarily stun mobs
+			D.stunned = 0 //Switching mind seems to temporarily stun mobs
 			for(var/mob/living/carbon/alien/diona/DIO in src.birds_of_feather) //its me!
 				DIO.master_nymph = D
 			break
-		return 1
+		return TRUE
 	..()
 
 /mob/living/carbon/proc/echo_eject()
 	set category = "Abilities"
 	set name = "Eject Echo"
 	set desc = "Eject any nymphatic husks."
 
-	if(stat == DEAD || paralysis || weakened || stunned || restrained())
-		return
+	if(use_check_and_message(usr))
+		return FALSE
 
 	var/energy
 
-	var/r = alert(src,"Do you choose to eject echoes as nymphs or energy?", "Identify Waste", "Energy", "Nymphs")
-	if (r == "Nymphs")
-		energy = 0
-	else if (r == "Energy")
-		energy = 1
+	var/r = alert(src, "Do you choose to eject echoes as nymphs or energy?", "Identify Waste", "Energy", "Nymphs")
+	if(r == "Nymphs")
+		energy = FALSE
+	else if(r == "Energy")
+		energy = TRUE
 	else
 		return
```

Comment:
Replace the above section with
```suggestion
		var/energy = FALSE
		var/r = alert(src, "Do you choose to eject echoes as nymphs or energy?", "Identify Waste", "Energy", "Nymphs")
		switch(r)
			if("Energy")
				energy = TRUE
			if("Identify Waste")
				return
```

Reply:
"Identify Waste" is the title of the alert